nestkernel/source_table.cpp

```diff
@@ -279,11 +279,11 @@ nest::SourceTable::source_should_be_processed_( const thread rank_start,
 {
   const thread source_rank = kernel().mpi_manager.get_process_id_of_node_id( source.get_node_id() );
 
-  return not( source.is_processed() or source.is_disabled()
-           // is this thread responsible for this part of the MPI
-           // buffer?
-           or source_rank < rank_start
-           or rank_end <= source_rank );
+  return not( source.is_processed()
+    or source.is_disabled()
+    // is this thread responsible for this part of the MPI
+    // buffer?
```

Comment:
Clang doesn't see/know that it can contract this comment. Probably it wouldn't break the single line:
```suggestion
    // is this thread responsible for this part of the MPI buffer?
```